sdk/python/express_relay/svm/limo_client.py

```diff
@@ -193,6 +193,9 @@ async def take_order_ix(
         input_mint_decimals: int,
         output_mint_decimals: int,
         express_relay_program_id: Pubkey,
+        permissionless_tip_lamports: int = 0,
```

Comment:
we actually don't want to expose these to searchers. it's more for testing without going through the auction server. So we can just remove it here